QMLFileType.js

```diff
@@ -126,7 +135,47 @@ QMLFileType.prototype.write = function(translations, locales) {
                     var manipulateKey = res.cleanHashKeyForTranslation(locale).replace(res.getContext(),"");
                     db.getResourceByCleanHashKey(manipulateKey, function(err, translated) {
                     var r = translated;
-                    if (!translated && customInheritLocale) {
+
+                    if (!translated && this.isloadCommonData) {
+                        var manipulateKey = res.cleanHashKeyForTranslation(locale).
+                                        replace("scrs_", "rs_").
+                                        replace(res.getContext()+"_", "").
+                                        replace("_"+ res.sourcehash, "").
+                                        replace(res.getProject(), this.commonPrjName).
+                                        replace("_" + res.getDataType() + "_", "_" + this.commonPrjType + "_");
```

Comment:
Maybe just do `SourceContextResourceString.hashKey(project, context, locale, reskey, datatype, flavor, sourcehash)`

Reply:
I updated it